src/infra/docs/rustc-ci.md

```diff
@@ -1,8 +1,16 @@
 # How the Rust CI works
 
+Rust CI ensures that the master branch of rust-lang/rust is always in a valid state.
+
+A developer submitting a pull request to rust-lang/rust, experiences the following:
+* A small subset of tests and checks are run on each commit to catch common errors.
```

Comment:
Nit: Can we stick to a consistent style for lists? The rest of the document seems to use `  -` as the prefix for lists. (Long-term, I really want to auto-format Markdown with Prettier 😬)